android/app/src/main/AndroidManifest.xml

```diff
@@ -62,6 +62,8 @@
         <data android:scheme="https" />
         <data android:host="deep.edge.app" />
         <data android:host="dl.edge.app" />
+        <data android:host="test.bitpay.com" />
```

Comment:
A similar change needs to be made in ios/edge/edge.entitlements

Reply:
Seems to work fine without any changes when simulating deep links via terminal or Safari